tests/agent/message_service/test_subscriber.py

```diff
@@ -0,0 +1,84 @@
+from unittest.mock import Mock, ANY
+
+import pytest
+
+from great_expectations.agent.message_service.subscriber import (
+    Subscriber,
+    SubscriberError,
+)
+from great_expectations.agent.models import Event
+from tests.agent.message_service.conftest import AMQP_CHANNEL_AND_CONNECTION_ERRORS
+
+
+def test_subscriber_has_client_attribute():
+    client = Mock()
+
+    subscriber = Subscriber(client=client)
+
+    assert subscriber.client is client
+
+
+def test_subscriber_consume_calls_basic_consume():
+    client = Mock()
```

Comment:
Can we use an auto-spec mock for this.

Reply:
updated in [866d2c0](https://github.com/great-expectations/great_expectations/pull/7837/commits/866d2c0ed473753180af6008a19680e4e48f5bf5)